### PR TITLE
Modify cohort_vcf_to_zarr.py to optionally output a zipped zarr.

### PR DESCRIPTION
**Pull request by gbggrant**
Modify cohort_vcf_to_zarr.py to optionally output a zipped zarr.
Pass that parameter (allow zip) to the cohort_vcf_to_zarr task in the WDL.
Improve the ability to pass runtime parameters to gcp tasks.

### CODE REVIEW

**Comment by gbggrant** [comment]
Still have duplicate copies of the python scripts in the 'scripts' directory and in the docker subdirectories (where they need to be for docker to work).  Okay to delete from the scripts directory @alimanfoo ?

**Review by jessicaway — APPROVED** [review]
looks good

File: pipelines/phasing-vector/farm5/Phasing.wdl [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    Array[String] chromosome_list
+    Array[File] genetic_maps
+
+    File? haplotype_reference_panel
+
+    ReferenceSequence reference
+    RunTimeSettings runTimeSettings
+  }
+
+  # TODO: extract sample_id, sample_bam, and sample_vcf information from the maifest file (or inputs)
```
Comment (alimanfoo) [inline pipelines/phasing-vector/farm5/Phasing.wdl:36]:
```suggestion
  # TODO: extract sample_id, sample_bam, and sample_vcf information from the manifest file (or inputs)
```

**Review by alimanfoo — APPROVED** [review]
LGTM. If the tests run then I think you're good here. Just picked up one typo in a comment.

**Pull request merged**